src/state.ts

```diff
@@ -208,6 +208,14 @@ export class FormState<
       this.updateFunc = options.update ? options.update : null;
       this._context = options.context;
       this._converterOptions = options.converterOptions || {};
+      if (
```

Comment:
Since you do this check in two places (and as I suggested elsewhere, you want to expand it), you can extract a function (for decimal.ts) that is called 'checkConverterOptions' that throws if there's something wrong, otherwise passes silently.